### DISCUSSION TITLE
Is there a limit in the number of environment per repository?

### DISCUSSION THREAD

**Discussion by cangulo** [community/community#53312] [Actions]
### Select Topic Area

Question

### Body

Hi there, 
I was checking the docs about the environments but I didn't find any mention to a limit. Is there a maximum of environments per repository? Thanks in advance :)

Here are the docs I read: https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment

**Comment by BenJackGill** [comment]
Did you ever figure this out? I have the same question

**Reply by cangulo** [reply]
no, I didn't. I just created a GH support ticket about this. If I get an answer I will post it here :)

**Comment by cangulo** [comment]
I got an answer from GH.

> There are currently no system-imposed limits to the number of environments you can have on any given repository meaning there as no such limits at the organization level as well.

FYI: @BenJackGill
